apps/nextjs-website/src/lib/api.ts

```diff
@@ -18,8 +18,8 @@ function manageUndefined<T>(props: undefined | null | T) {
   return props;
 }
 
-function manageUndefinedAndAddProduct<T>(props: undefined | null | T) {
-  return { ...manageUndefined(props), products: [...getProducts()] };
+async function manageUndefinedAndAddProduct<T>(props: undefined | null | T) {
+  return { ...manageUndefined(props), products: [...(await getProducts())] };
```

Comment:
```suggestion
  return { ...manageUndefined(props), products: await getProducts() };
```
same, you don't need to destructure the array